pkg/forwarder/forwarder.go

```diff
@@ -0,0 +1,80 @@
+package forwarder
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/storageos/kubectl-storageos/pkg/logger"
+	pluginutils "github.com/storageos/kubectl-storageos/pkg/utils"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	errCLIPodUnavailable = "storageos cli pod unavailable for request - please ensure storageos-cli deployment is running"
+	cliLabel             = "app.kubernetes.io/component=storageos-cli"
+)
+
+type forwarder struct {
+	clientConfig *rest.Config
+	log          *logger.Logger
+}
+
+func NewForwarder(log *logger.Logger) (*forwarder, error) {
+	clientConfig, err := pluginutils.NewClientConfig()
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+	log.Verbose = true
```

Comment:
This looks a side effect.

Reply:
Each command creates its own instance of `logger` so it won't impact other commands. And for these commands it is only used to print the output of the underlying CLI command. So we never actually log anything of our own (except for errors which ignore verbosity).